SPECS-EXTENDED/python-enchant/python-enchant.spec

```diff
@@ -0,0 +1,311 @@
+%global srcname enchant
+
+Name:           python-enchant
+Version:        3.2.2
+Release:        7%{?dist}
+Summary:        Python bindings for Enchant spellchecking library
+License:        LGPLv2+
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            http://packages.python.org/pyenchant/
```

Comment:
This URL throws a 404 :(
can we put the github location ? for both url & source ?